src/metabase/query_processor/middleware/results_metadata.clj

```diff
@@ -49,7 +49,7 @@
      (merge
       (select-keys final-col [:id :description :display_name :semantic_type :fk_target_field_id
                               :settings :field_ref :name :base_type :effective_type
-                              :coercion_strategy :semantic_type])
+                              :coercion_strategy :semantic_type :remapped_from :remapped_to])
```

Comment:
Is `remapped_to` here for symmetry reasons?

Reply:
yes.